roles/ceph-mon/tasks/docker/main.yml

```diff
@@ -94,8 +94,12 @@
     - not containerized_deployment_with_kv
 
 - block:
+  - name: get mons ips addr
+    command: "{{ docker_exec_cmd }} ceph --cluster {{ cluster }} ceph-conf -c {{ ceph_conf_key_directory }}/{{ cluster}}.conf 'mon host'"
```

Comment:
Nop :p 

`command: "{{ docker_exec_cmd }} ceph-conf -c {{ ceph_conf_key_directory }}/{{ cluster}}.conf 'mon host'"`

is what you want.